index.html

```diff
@@ -1,14 +1,79 @@
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet"> 
+  <script src="https://cdn.tailwindcss.com"></script>
+  <script src="index.js"></script>
   <title>andkit</title>
 </head>
-<body>
-  <header>
+<body class="bg-gray-100">
+  <header class="bg-white py-2">
+    <div class=" container mx-auto max-w-7xl px-4 sm:px-6">
+      <div class="flex h-16 items-center gap-72">
+        <div class="flex items-center">
+          <a href="#"><svg width="111" height="43" viewBox="0 0 111 43" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
No hay un indicador de que es este link ni en el link, ni en el svg
```suggestion
          <a href="#" aria-label="Go to home page"><svg width="111" height="43" viewBox="0 0 111 43" fill="none" xmlns="http://www.w3.org/2000/svg" aria-label="Site logo">
```